src/NServiceBus.Core/Performance/TimeToBeReceived/ApplyTimeToBeReceivedBehavior.cs

```diff
@@ -22,7 +21,7 @@ public override Task Invoke(OutgoingLogicalMessageContext context, Func<Task> ne
             if (timeToBeReceivedMappings.TryGetTimeToBeReceived(context.Message.MessageType, out timeToBeReceived))
             {
                 context.AddDeliveryConstraint(new DiscardIfNotReceivedBefore(timeToBeReceived));
-                context.SetHeader(Headers.TimeToBeReceived, timeToBeReceived.ToString());
+                context.Headers[Headers.TimeToBeReceived] = timeToBeReceived.ToString();
```

Comment:
Is this safe? Shouldn't we use a well defined format? It would use the culture..